agr_indexer/src/main/java/org/alliancegenome/indexer/config/IndexerConfig.java

```diff
@@ -23,6 +25,8 @@ public enum IndexerConfig {
 	GoIndexer("go", GoIndexer.class, 4, 914, 914, 8, 1),				// (44087,	262, 460247, 1093)
 	ModelIndexer("model", ModelIndexer.class, 4, 1426, 1426, 8, 1),		   // (132447, 314,	 18593,	 701)
 	DiseaseAnnotationMlIndexer("diseaseAnnotation", DiseaseAnnotationCurationIndexer.class, 1, 1426, 1426, 2, 1),
+	GeneGeneticInteractionIndexers("geneGeneticInteraction", GeneGeneticInteractionCurationIndexer.class, 1, 1426, 1426, 2, 1),
+	GeneMolecularInteractionIndexers("geneMolecularInteraction", GeneMolecularInteractionCurationIndexer.class, 1, 1426, 1426, 2, 1),
```

Comment:
You might want to change the thread count here to 4 and set the buffer to a more whole number.

Reply:
Updated